soh/soh/OTRGlobals.cpp

```diff
@@ -230,6 +231,11 @@ OTRGlobals::OTRGlobals() {
             }
         }
     }
+        std::string sohAccessibilityPath = LUS::Context::GetPathRelativeToAppDirectory("accessibility.otr");
```

Comment:
curious as to how people feel about this vs. adding stuff to `soh.otr` in an accessibility directory

Reply:
Yeah, if it's going to be merged in, keeping the official stuff in a single OTR would be my preference.

Reply:
I'm also in favour of having everything undeer the soh.otr and we're already doing that for the TTSwhere things goes under accessibility :)